index.html

```diff
@@ -47,10 +47,154 @@ <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
         </ul>   
       </div>
       
-      <div class="arrow">
+    </section>
+
+    <section class="recent-works"> <!-- flexbox container-->
+      <div class="arrow"> <!-- flexbox item-->
         <i class="fa-solid fa-circle-chevron-down"></i>
       </div>
+
+      <div class="title"> <!-- flexbox item-->
+        <p>My Recent Works</p>
+      </div>
+
+      <div class="bar"></div> <!-- flexbox item-->
+
+      <div class="recent-works-cards">  <!-- flexbox item-->
+        <div class="works work1"> <!-- flexbox container-->
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
```

Comment:
- [ ] Referencing the project requirements, each project title should use a header tag` hx` (h1, h2, h3,..) . Please, change the` p` tag with a` hx` tag of your choice.